packages/components/multiselect/stories/Multiselect.stories.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import type { Meta } from '@storybook/react/types-6-0';
+
+import { Stack } from '@contentful/f36-core';
+import { Multiselect, MultiselectProps } from '../src';
+
+export default {
+  title: 'Components/Multiselect',
+  component: Multiselect,
+} as Meta;
+
+interface Produce {
+  id: number;
+  name: string;
+}
+
+const fruits: Produce[] = [
+  { id: 1, name: 'Apple 🍎' },
+  { id: 2, name: 'Pineapple 🍍' },
+  { id: 3, name: 'Avocado 🥑' },
+  { id: 4, name: 'Banana 🍌' },
+  { id: 5, name: 'Coconut 🥥' },
+  { id: 6, name: 'Lemon 🍋' },
+  { id: 7, name: 'Orange 🍊' },
+  { id: 8, name: 'Peach 🍑' },
+  { id: 9, name: 'Pear 🍐' },
+  { id: 10, name: 'Strawberry 🍓' },
+  { id: 11, name: 'Tangerine 🍊' },
+  { id: 12, name: 'Tomato 🍅' },
+];
+
+const fruitStrings = fruits.reduce((acc, fruit) => [...acc, fruit.name], []);
+
+export const Basic = (args: MultiselectProps) => {
+  const [selectedFruits, setSelectedFruits] = useState<Array<string>>([]);
+
+  const handleSelectItem = (event) => {
+    const { checked, value } = event.target;
+    if (checked) {
+      setSelectedFruits((prevState) => [...prevState, value]);
+    } else {
+      const newSelectedFruits = selectedFruits.filter(
+        (fruit) => fruit !== value,
+      );
+      setSelectedFruits(newSelectedFruits);
+    }
+  };
+
+  return (
+    <Stack
+      style={{ width: '150px' }}
+      flexDirection="column"
+      spacing="spacingM"
+      alignItems="start"
+    >
+      <Multiselect {...args} currentSelection={selectedFruits}>
+        {fruitStrings.map((item, index) => {
+          return (
+            <Multiselect.Option
+              value={item}
+              label={item}
+              onSelectItem={handleSelectItem}
+              key={`${item}-${index}`}
```

Comment:
It's just an example code but still I would like to leave the feedback that key shouldn't use the index if possible:

See the [official docs](https://reactjs.org/docs/lists-and-keys.html).

> We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state. Check out Robin Pokorny’s article for an [in-depth explanation on the negative impacts of using an index as a key](https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/). If you choose not to assign an explicit key to list items then React will default to using indexes as keys.